lfucg-exactions/client/js/components/AccountLedgerForm.js

```diff
@@ -275,6 +275,63 @@ class AccountLedgerForm extends React.Component {
                                 </div>
                             </div>
                             }
+                            <div className="clearfix" />
+                            {activeForm.lot || activeForm.plat ? <div>
+                                {
+                                    selectedAccountLedger ?
+                                        lots && lots.length > 0 &&
+                                            map((lot =>
+                                                (lot.id === parseInt(activeForm.lot, 10)) ?
+                                                    <Notes
+                                                      secondary_content_type="plats_lot"
+                                                      secondary_object_id={lot.id}
+                                                      content_type={selectedAccountLedger && 'accounts_accountledger'}
+                                                      object_id={selectedAccountLedger && selectedAccountLedger}
```

Comment:
selectedAccountLedger is not None at this point since you checked it in the ternary above.